src/MagnumPlugins/AssimpImporter/Test/image-filename-non-normalized.mtl

```diff
@@ -0,0 +1,13 @@
+newmtl Material
+Ns 96.078431
+Ka 1.0 1.0 1.0
+Kd 0.64 0.64 0.64
+Ks 0.5 0.5 0.5
+Ke 0.0 0.0 0.0
+Ni 1.0
+d 1.0
+illum 2
+# There's a space here ---v
```

Comment:
Leftover comment copied from the other file that tested trailing whitespace in filenames? :)